csvcubed/tests/behaviour/steps/qbwriter.py

```diff
@@ -145,7 +145,7 @@ def step_impl(context, cube_name: str, csvw_file_path: str):
     ]
 
     csv_path, _ = get_first_table_schema(csvw_path)
-    code_list_data = read_csv(csv = csvw_path.parent / csv_path)
+    code_list_data, _data_errors = read_csv(csv=csvw_path.parent / csv_path)
```

Comment:
Again, we should be testing something about these errors. Ideally they should be stored in the context and validated in the `the CSVqb should pass all validations` step.